rpm-build/SPECS/securedrop-workstation-dom0-config.spec

```diff
@@ -22,17 +22,23 @@ SecureDrop Workstation project. The package should be installed
 in dom0, or AdminVM, context, in order to manage updates to the VM
 configuration over time.
 
+# To ensure forward-compatibility of RPMs regardless of updates to the system
+# Python, we disable the creation of bytecode at build time via the build
+# root policy.
+%undefine py_auto_byte_compile
```

Comment:
This is the change discussed in https://github.com/freedomofpress/securedrop-workstation/issues/449#issuecomment-584484190 .